src/main/java/de/tum/in/www1/artemis/web/rest/TeamResource.java

```diff
@@ -317,12 +320,54 @@ public TeamResource(TeamRepository teamRepository, TeamService teamService, Team
         return ResponseEntity.ok().body(teamService.searchByLoginOrNameInCourseForExerciseTeam(course, exercise, loginOrName));
     }
 
+    /**
+     * PUT /exercises/:destinationExerciseId/teams/import-from-file : add given teams into exercise
+     *
+     * @param exerciseId         the exercise id of the exercise for which to import teams
+     * @param teams              teams whose students have login or registration number as identifiers
+     * @param importStrategyType the import strategy to use when importing the teams
+     * @return the ResponseEntity with status 200 (OK) and the list of created teams in body
+     */
+    @PutMapping("/exercises/{exerciseId}/teams/import-from-list")
```

Comment:
why is this a PUT mapping? If we add new objects into the database, should we not use POST instead?

Reply:
I believe PUT is a valid option here since there is a possibility that it deletes the existing teams. So it is more like updating exercise's teams. It was also a put request for importing from exercise.